### DISCUSSION TITLE
how to commit with .github dir?

### DISCUSSION THREAD

**Discussion by pplmx** [peaceiris/actions-gh-pages#777] [Q&A] [answered]
My Scenario:

1. cargo-generate-action generate a project with .github dir 
2. I want to push this project with **all files** to another repo

Could anyone give me some help? Thanks a lot.

**Comment by pplmx** [comment, marked as answer]
And also I don't need `.nojekyll`, Is anyway to prevent generating it?

**Reply by peaceiris** [reply]
Please see the README first.

- https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-filter-publishing-assets-exclude_assets
- https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-enable-built-in-jekyll-enable_jekyll

**Reply by pplmx** [reply]
I tried these:

- `exclude_assets: '.nojekyll'`
- `enable_jekyll: false`

`.github` dir can be included, but `.nojekyll` will still be generated

**Reply by peaceiris** [reply]
enable_jekyll: true
